src/components/notes.js

```diff
@@ -0,0 +1,47 @@
+import "../App.css";
+import { useState } from "react";
+
+const Note = () => {
+  const [show, setShow] = useState(false);
+  const [note, setNote] = useState();
+  const [notesArray, setnotesArray] = useState([]);
+  const clickHandler = (e) => {
+    if (show !== true) {
+      setShow(true);
+    } else {
+      setShow(false);
+    }
+  };
+
```

Comment:
if you want to switch the boolean based on the previous it can be done like this 
`setShow(show => !show)`

Reply:
okay